freeswitch/scripts/astpp/lib/astpp.functions.lua

```diff
@@ -281,6 +281,18 @@ function do_number_translation(number_translation,destination_number)
       if(tmp_str[1] == '' or tmp_str[1] == nil)then
 	return destination_number
       end
+
+      if (string.sub(tmp_str[1],0,3) == 'LP:') then
```

Comment:
Hi @mihalevm @il-s 

Thanks for your contribution. 

We would like to understand this condition code. Can you please explain with example? 
I appreciate your efforts.

Reply:
Example in request text:
"For example, you want to check that the number contains 11 digits and starts at 79 (RUSSIA mobile), then writing in the CUT field: "LP: ^(79.........)$", in the ADD field: "888%1", we can add a special prefix for routing. Checking the length will save us from problems if an internal number starts at 79 or for example a local number (city numbers of 6 digits)."

The prefix "LP:" (Lua Pattern) allows you to separate the normal behavior during translation and translation using regular expressions.